src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -151,13 +161,21 @@
      */
     private static final int MAX_NUM_TOKENS = 1536;
 
+    /**
+     * The metadata source of configuration properties. It is used to access the configuration class instance loaded
+     * from the {@link #conf} and must be initialized the same time the {@link #conf} is initialized, usually
+     * by calling {@link #toolInitialization()}, {@link #daemonInitialization()} or {@link #clientInitialization()}.
+     */
+    private static Map<String, ? extends Property> confValueAccessors;
     private static Config conf;
 
     /**
      * Request timeouts can not be less than below defined value (see CASSANDRA-9375)
      */
     static final DurationSpec.LongMillisecondsBound LOWEST_ACCEPTED_TIMEOUT = new DurationSpec.LongMillisecondsBound(10L);
 
+    public static final int SPACE_UPPER_BOUND_MB = (int) (Runtime.getRuntime().maxMemory() / (16 * 1048576));
```

Comment:
`private`

Reply:
This was used in tests, but along with removing support for the delete operation, I made it private as you suggested.